gulpfile.js

```diff
@@ -31,7 +32,8 @@ gulp.task('clean', function(cb) {
 
 
 gulp.task('src', function() {
-    var project = typescript.createProject({
+    var project = gulpTypescript.createProject({
+        typescript: typescript,
```

Comment:
suggest adding support for experimental decorators, which is required to declare Phosphor signals using the nice decorator syntax:

https://github.com/phosphorjs/phosphor/blob/master/gulpfile.js#L133